driver-service/src/main/java/com/example/driverservice/mapper/CarMapper.java

```diff
@@ -2,43 +2,43 @@
 
 import com.example.driverservice.dto.car.CarRequest;
 import com.example.driverservice.dto.car.CarResponse;
-import com.example.driverservice.dto.car.CarResponseList;
+import com.example.driverservice.dto.car.CarPageResponse;
 import com.example.driverservice.model.Car;
 import com.example.driverservice.model.Driver;
 import org.mapstruct.Mapper;
 import org.mapstruct.Mapping;
+import org.mapstruct.MappingTarget;
 import org.mapstruct.Named;
+import org.mapstruct.ReportingPolicy;
+import org.springframework.data.domain.Page;
 
 import java.util.List;
 
-@Mapper(componentModel = "spring")
-public abstract class CarMapper {
-    @Mapping(target = "driverId", source = "driver", qualifiedByName = "driverToDriverId")
-    public abstract CarResponse toResponse(Car car);
+@Mapper(componentModel = "spring", unmappedTargetPolicy = ReportingPolicy.IGNORE)
```

Comment:
```suggestion
@Mapper(componentModel = MappingConstants.ComponentModel.SPRING, unmappedTargetPolicy = ReportingPolicy.IGNORE)
```

чисто во избежание описок/ошибок